libscript/src/assert.lcb

```diff
@@ -61,12 +61,12 @@ may require that its parameters have a particular structure or fall
 within a particular range, or may need the system to be in a specific
 state.
 
-Note that if the <Condition> is false, this statement will throw an
```

Comment:
@PaulMcClernan the issue is in the `Parameters` section there is a tab before `Condition`. It looks like the same issue is in the `ExpectPreconditionWithReason` docs below.